src/xrGame/DiscordRPC.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+class DiscordRPC
+{
+    int64_t start_time{};
+    string64 build_name{};
+    string256 current_level_name{};
+    string256 current_task_name{};
+
+public:
+    void Init();
+    void Deinit();
+
+    void Update(bool type, pcstr data);
```

Comment:
I don't like the name of the first argument. It's not clear what type or how should we use it.
According to the usage of this argument, I'd suggest to rename it to smth like `isLevelUpdate` or similar.

cc: @Xottab-DUTY

Reply:
Ok, let's update this as we discussed offline. I'll mark other related comments as resolved for now.